sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/events/CreateNewEventSteps.java

```diff
@@ -41,10 +46,146 @@ public CreateNewEventSteps(WebDriverHelpers webDriverHelpers) {
           webDriverHelpers.fillInWebElement(
               TITLE_INPUT, "EVENT_AUTOMATION" + timestamp + Faker.instance().name().name());
           webDriverHelpers.clickOnWebElementBySelector(SAVE_BUTTON);
-          webDriverHelpers.waitUntilElementIsVisibleAndClickable(ADD_PARTICIPANT_BUTTON);
+          webDriverHelpers.waitUntilElementIsVisibleAndClickable(NEW_EVENT_CREATED_MESSAGE);
```

Comment:
You don't need to add this because is already called on next line. inside `clickOnWebElementBySelector(NEW_EVENT_CREATED_MESSAGE)`